src/gateway_config.erl

```diff
@@ -65,6 +65,8 @@ wifi_services() ->
                                        {true, {Name, Strength}};
                                   ({_Path, _}) -> false
                          end, connman:services()),
+    %% Start another scan for visible wifi services
+    connman:scan(wifi),
```

Comment:
I don't think this helps here if you already trigger one when advertising is enabled?

Reply:
I think you're right about that.  I added this scan before the one on advertising.  It's probably unnecessary now.

Reply:
Oddly the button only seems to trigger a Wi-Fi scan the first time its pressed.  Subsequent button presses (after the blue LED turns green) do not repopulate the list.  I also noticed that the list empties out after a while (less than the 5 minute BLE advertising timeout).  Did I put the Wi-Fi scan on advertising enabled in the wrong place?

Reply:
I did.  Only the CLI calls `advertising_enable(Enable)`.  The button does not.